paparazzi/paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -116,7 +116,7 @@ class Paparazzi @JvmOverloads constructor(
 
   override fun apply(
     base: Statement,
-    description: Description
+    description: Description,
```

Comment:
hmm, is your spotless cache or ide inspection settings corrupt?  we haven't bumped spotless recently, so these suggestions shouldn't be happening.  let's revert?